exchanges/credentials.go

```diff
@@ -117,29 +117,30 @@ func (b *Base) GetCredentials(ctx context.Context) (*account.Credentials, error)
 		if !ok {
 			// NOTE: Return empty credentials on error to limit panic on
 			// websocket handling.
-			return &account.Credentials{}, errContextCredentialsFailure
+			return nil, fmt.Errorf("context credentials store type assertion failed for %T: %w", value, errContextCredentialsFailure)
```

Comment:
For type assertions we have a nice happy little helper `common.GetTypeAssertError()` which wraps whats expected and what was received which you can use here.